pkg/asset/manifests/template.go

```diff
@@ -6,6 +6,17 @@ type AwsCredsSecretData struct {
 	Base64encodeSecretAccessKey string
 }
 
+// AzureCredsSecretData holds encoded credentials and is used to generate cloud-creds secret
```

Comment:
also there are no consumers for this yet?
https://github.com/openshift/installer/blob/0137c5460221c26ee7d966014c907372926188d1/data/data/manifests/openshift/cloud-creds-secret.yaml.template
https://github.com/openshift/installer/blob/0137c5460221c26ee7d966014c907372926188d1/data/data/manifests/openshift/role-cloud-creds-secret-reader.yaml.template

Reply:
because the manifests are in data/data and will come in a different PR